src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/AzureADTokenRequestContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace Microsoft.Data.SqlClient
+{
+    /// <summary>
+    ///
+    /// </summary>
+    public class AzureADTokenRequestContext
```

Comment:
Good point on not using AAD. AccessTokenRequestContext might be even better here. We don't have the term "Azure" in any of our public APIs.